Sources/Plasma/FeatureLib/pfDXPipeline/plDXPipeline.cpp

```diff
@@ -3337,6 +3337,7 @@ bool  plDXPipeline::CaptureScreen( plMipmap *dest, bool flipVertical, uint16_t d
     }
     else
     {
+        // FIXME: DPI awareness
```

Comment:
`fDevice.fHwnd` is the HWND for the client window, if that's useful here

Reply:
This function is never used anywhere, so omitting should be fine for now, IMO.